mix/CodeModel.h

```diff
@@ -64,6 +64,29 @@ public slots:
 	CodeModel* m_model;
 };
 
+using LocationPair = QPair<int, int>;
+
+struct SolidityType
```

Comment:
Do you really want to create new structs there that are needed to be kept in sync with libsolidity/Types.h?